### PR TITLE
Support a MAVEN_CMD env to configure the maven binary used to build the project.

### PR DESCRIPTION
**Pull request by chirino**

### CODE REVIEW

**Comment by rhuss** [comment]
retest this please

**Comment by iocanel** [comment]
@rhuss: does `retest this please` also work for circleci?

**Comment by rhuss** [comment]
@iocanel :  oops, of course not. Let me trigger it manually ...

**Comment by rhuss** [comment]
ehm, the circle build is not ready yet as it looks like. We should probably make it not mandatory.

File: build.sh [hunk 1]
```diff
@@ -73,9 +73,10 @@ function init_options() {
 
   # Internal variable default values
   OC_OPTS=""
-  MAVEN_OPTS=""
+  MAVEN_OPTS="--batch-mode"
```
Comment (rhuss) [inline build.sh]: Can't we set batch-mode only when running in a CI ? Having it as default doesn't feel right to me (I like colors ;-)

Comment (chirino) [reply build.sh]: true.

**Pull request merged**